settings.py

```diff
@@ -103,6 +105,27 @@ def get_flask_template_path() -> str:
 else:
   logging.error('Unexpected app ID %r, please configure settings.py.', APP_ID)
 
+def get_ot_api_key() -> str|None:
+  # In dev or unit test mode, pull the API key from a local file.
+  if DEV_MODE or UNIT_TEST_MODE:
+    try:
+      with open(f'{ROOT_DIR}/ot_api_key.txt', 'r') as f:
```

Comment:
Can you add something to the README about this file and how to generate it

Reply:
I'm curious how we want to handle this for a new contributor actually. This is just a workaround for the user not having access to the GCP project secrets locally, and is a text file containing the API key. So there wouldn't really be a way to generate this file, and it would need to be obtained by someone who already has access to the key. Is there a better way to go about this? It's a problem I'm not sure the best solution for.

Reply:
Locally, I think it's okay that a new contributor doesn't have access to it for now.
These instructions would be for us.  You could add a new package.json command that does the [download](https://cloud.google.com/sdk/gcloud/reference/secrets/versions/access) of the staging secret from GCP. `gcloud secrets versions access --out-file=thefile.txt --project=cr-status-staging`. The instructions could say something like:

```md
# Origin Trials
To test Origin Trial functionality locally, you will need to run the following: `<insert command>`

Note: *Only developers with access to the cr-status-staging project will be able to run this command. If you need to test this and you don't have access, open a bug.*
```

Additionally, you may want to generate a separate secret for staging for the devs. One secret that will be used strictly for the server to talk to OT staging, the other one to be used locally by devs and their servers to talk to OT staging

Reply:
I'd rather not add a lot of imperative code to settings.py.   How about specifying `OT_API_URL` here, but putting the code to access the OT API key in the existing file `framework/secrets.py` or putting it in a new file named something like`internals/otconsole.py`?